tutorials/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -121,6 +85,28 @@ external editors:
     If you are using Visual Studio 2019, you must follow the instructions found
     in the `:ref:doc_c_sharp_configuring_vs_2019_for_debugging` section below.
 
+Windows (Visual Studio)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Download and install the latest version of
+`Visual Studio <https://visualstudio.microsoft.com/downloads/>`_
+(*not* Visual Studio Code), which contains utilities required to use
+C# in Godot.
```

Comment:
Is this actually required to install Visual Studio? In my experience just Mono and VS Code work. Although I haven't tested on Windows without Visual Studio in awhile, I don't plan to uninstall Visual Studio to test this.

If the tools included in Visual Studio are required, we may want to change this text around so that "JetBrains Rider" is a sub-heading.

Reply:
No, visual studio is not a must have. I have just moved it and JetBrains rider down inside 'Configuring an external editor'. Take a look at the whole file, it makes more sense now, I think.

Reply:
Did you forget to push it?

Reply:
I don't mean I made any additional changes today, sorry for confusion.